### ISSUE TITLE
Spring Boot Build-Image no longer works on GCP Cloud Build

### ISSUE THREAD

**Issue by westmc** [paketo-buildpacks/base-builder#143]
All of our Spring Boot image builds stopped working today. We tracked it down to the use of paketobuildpacks/builder:base. The process hangs indefinitely on the "Running creator" step as of 1.0.87-base. The same problem occurs with 1.0.88-base. Manually configuring Spring Boot to use 1.0.86-base fixes the problem.

Command: mvn clean spring-boot:build-image
Environment: GCP Cloud Build
Spring Boot Versions: 2.4.3 and 2.4.4
Log:

```
Step #1 - "build": Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-classworlds/2.6.0/plexus-classworlds-2.6.0.jar (53 kB at 102 kB/s)
Step #1 - "build": Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.2.1/plexus-utils-3.2.1.jar (262 kB at 497 kB/s)
Step #1 - "build": [INFO] Replacing main artifact with repackaged archive
Step #1 - "build": [INFO] 
Step #1 - "build": [INFO] <<< spring-boot-maven-plugin:2.4.3:build-image (default-cli) < package @ XXX <<<
Step #1 - "build": [INFO] 
Step #1 - "build": [INFO] 
Step #1 - "build": [INFO] --- spring-boot-maven-plugin:2.4.3:build-image (default-cli) @ XXX ---
Step #1 - "build": [INFO] Building image 'gcr.io/XXX/XXX:develop'
Step #1 - "build": [INFO] 
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 6%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 15%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 21%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 21%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 24%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 28%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 36%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 44%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 51%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 56%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 60%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 64%
Step #1 - "build": [INFO]  > Pulling builder image 'docker.io/paketobuildpacks/builder:base' 100%
Step #1 - "build": [INFO]  > Pulled builder image 'paketobuildpacks/builder@sha256:f9035642ff8934e1eaf54661ef4975efd5050ab8b38315cc64434ca8a5d771e5'
Step #1 - "build": [INFO]  > Pulling run image 'docker.io/paketobuildpacks/run:base-cnb' 100%
Step #1 - "build": [INFO]  > Pulled run image 'paketobuildpacks/run@sha256:0bf521429c5fac06616ef542da735f9e34c4997cc5d5987242eb7199b04ac923'
Step #1 - "build": [INFO]  > Executing lifecycle version v0.11.0
Step #1 - "build": [INFO]  > Using build cache volume 'pack-cache-3482662deee0.build'
Step #1 - "build": [INFO] 
Step #1 - "build": [INFO]  > Running creator
```

**Comment by fg-j** [comment]
Hi there. Thanks for raising this. 

This is a problem with CNB lifecycle version 0.11.0 ([this issue](https://github.com/buildpacks/lifecycle/issues/558)) is tracking the bug. The reason you saw this is because base builder versions 0.1.87 and 0.1.88 were released with that version of the lifecycle (see our release notes). We've since rolled back the version of the lifecycle in the builders to 0.10.2 until this bug is resolved.
[Base builder version 0.1.89](https://github.com/paketo-buildpacks/base-builder/releases/tag/v0.1.89) has lifecycle version 0.10.2. Pulling the latest builder should fix this problem for you 🙂
